src/views/PostCreator/components/PostTilePreview/PostTilePreview.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Accordion, AccordionSummary, AccordionDetails, Typography } from '@material-ui/core';
+import { ExpandMore } from '@material-ui/icons';
+import { PostTile } from 'components';
+
+/* eslint-disable no-useless-escape */
```

Comment:
tu chyba jak się da te disable to potem gdzieś trzeba wrzucić enable

Reply:
A to nie działa tylko w tym pliku?